### PR TITLE
Added support ToxicDog69 PCB

### PR DESCRIPTION
**Pull request by HorrorTroll**
## Description

Added support ToxicDog69 PCB, using RP2040 MCU

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/horrortroll/toxicdog69/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+#include QMK_KEYBOARD_H
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+
+enum layer_names {
+    _BASE = 0,
+    _FN = 1
```
Comment (drashna) [inline keyboards/horrortroll/toxicdog69/keymaps/default/keymap.c:26]: you don't need to explicitly set the values.  Enums start at 1 and automatically increment up.
```suggestion
    _BASE,
    _FN,
```

Comment (HorrorTroll) [reply keyboards/horrortroll/toxicdog69/keymaps/default/keymap.c]: I've just resolved those issues, thanks!

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Retargeting master, since that has RP2040 support now.

File: keyboards/horrortroll/toxicdog69/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+
+enum layer_names {
+    _BASE,
+    _FN,
+};
+
+// enum layer_keycodes {}
```
Comment (zvecr) [inline keyboards/horrortroll/toxicdog69/keymaps/default/keymap.c:29]:
```suggestion
```

File: keyboards/horrortroll/toxicdog69/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+
+enum layer_names {
+    _BASE,
+    _FN,
+};
+
+// enum layer_keycodes {}
```
Comment (zvecr) [inline keyboards/horrortroll/toxicdog69/keymaps/via/keymap.c:29]:
```suggestion
```

File: keyboards/horrortroll/toxicdog69/rules.mk [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+MCU = RP2040
+
+# Bootloader selection
+BOOTLOADER = rp2040
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no        # Mouse keys
```
Comment (zvecr) [inline keyboards/horrortroll/toxicdog69/rules.mk:11]:
```suggestion
MOUSEKEY_ENABLE = yes       # Mouse keys
```

File: keyboards/horrortroll/toxicdog69/keymaps/via/config.h [hunk 5]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2022 HorrorTroll <https://github.com/HorrorTroll>
```
Comment (zvecr) [inline keyboards/horrortroll/toxicdog69/keymaps/via/config.h:1]: This file should be removed

**Review by zvecr — CHANGES_REQUESTED** [review]

**Comment by HorrorTroll** [comment]
Closed due to not active for this one anymore

**Pull request closed without merging**